docs/UserGuide.md

```diff
@@ -237,15 +237,16 @@ Adds a student to TutorBuddy.
 Format: `add_student n/NAME p/STUDENT_PHONE_NUMBER e/EMAIL a/ADDRESS l/STUDY_LEVEL g/GUARDIAN_PHONE_NUMBER r/RELATIONSHIP_WITH_GUARDIAN` <br>
 
 <div markdown="block" class="alert alert-info">
-:information_source: <code>STUDENT_PHONE_NUMBER</code>, <code>GUARDIAN_PHONE_NUMBER</code> should be in Singapore's phone formatting (i.e. starting with either 6, 8 or 9 and with 8 digits)
+:information_source: <code>STUDENT_PHONE_NUMBER</code>, <code>GUARDIAN_PHONE_NUMBER</code> should be in Singapore's phone formatting (i.e. starting with either 6, 8 or 9 and with 8 digits).
+:information_source: TutorBuddy automatically shows the full list of students and sessions after a successful <code>add_student</code> command.
 </div>
 
 Example:
 * `add_student n/John Doe p/98765432 e/johnd@example.com a/John street, Block 123, #01-01 l/Sec2 g/95421323 r/Mother`
 
 ##### Locating student by name: `find_student`
 
-Find student(s) whose names contain any of the given keywords.
+Filters the list to only include students, and their respective session whose student name contains any of the given keywords.
```

Comment:
On line 249,
"Filters the **student and session** list to only include **student(s)**, and their respective **session(s)** whose student name contains any of the given keywords."

Reply:
Right, this sounds better! Have updated it in the latest commit.